### PR TITLE
feat(snapchat-web): init

### PR DESCRIPTION
**Pull request by itzTheMeow**
## 🎉 Theme for Snapchat Web 🎉

Theme for snapchat's web client.
image
image
image

## 💬 Additional Comments 💬

can't really theme this section (or card) since theres no decent selector
the card is just transparent with a blur effect, background image is generated
image

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

File: scripts/userstyles.yml [hunk 1]
```diff
@@ -102,6 +102,9 @@ collaborators:
   - &rooot
     name: rooot
     url: https://github.com/RoootTheFox
+  - &itzTheMeow_
```
Comment (uncenter) [inline scripts/userstyles.yml:105]: Extra underscore? Not sure why that's here...
```suggestion
  - &itzTheMeow
```

Comment (itzTheMeow) [reply scripts/userstyles.yml]: oops sorry, used to typing the _

File: styles/snapchat-web/catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
+/* ==UserStyle==
+@name Snapchat Web Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/snapchat-web
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/snapchat-web
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/snapchat-web/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Asnapchat-web
+@description Soothing pastel theme for Snapchat Web
+@author Meow
```
Comment (isabelroses) [inline styles/snapchat-web/catppuccin.user.css:9]:
```suggestion
@author Catppuccin
```

File: styles/snapchat-web/catppuccin.user.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      --sigBackgroundPrimary: @base;
+      --sigBackgroundSecondary: @mantle;
+      --sigBackgroundSecondaryHover: darken(@mantle, 5%);
+      --sigBackgroundTertiary: @crust;
+      --sigBackgroundMessageHover: lighten(@crust, 7%);
+      --sigOverlay: fade(@overlay0, 0.4);
+      --sigOverlayHover: fade(@overlay0, 0.35);
+
+      --sigTextPrimary: @text;
+      --sigButtonOnPrimary: @text;
```
Comment (isabelroses) [inline styles/snapchat-web/catppuccin.user.css:77]:
```suggestion
      --sigButtonOnPrimary: @crust;
```

This should fix the poor contrast here 
image

File: styles/snapchat-web/catppuccin.user.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+      --sigIconCalling: @green;
+      --sigIconCallingHover: darken(@green, 5%);
+      --sigChatSurfaceCalling: @green;
+      --sigChatSurfaceCallingDisabled: darken(@green, 20%);
+
+      --sigChat: @blue;
+      --sigChatIcon: @blue;
+      --sigSnapWithoutSound: @red;
+      --sigSnapWithSound: @mauve;
+    }
```
Comment (isabelroses) [inline styles/snapchat-web/catppuccin.user.css:106]:
```suggestion
    }
    
    path[fill="#fff"] {
      fill: @crust;
    }
```

This should fix the poor contrast on similar buttons to this one:
image

**Review by isabelroses — CHANGES_REQUESTED** [review]

**Comment by itzTheMeow** [comment]
also styled this icon
image

--sigBrandSecondary doesnt appear to be used elsewhere

**Comment by itzTheMeow** [comment]
i missed more stuff so

buttons when in call
image
empty checkbox border
image
selected checkbox in new chat menu (these use a hardcoded svg color instead of the other ones which are the "secondary brand" color...)
image
new group / new chat button content
image

**Review by isabelroses — APPROVED** [review]
LGTM, thanks for making all the appropriate changes. It looks amazing.

**Pull request merged**